### DISCUSSION TITLE
cleanepi: Package for data cleaning

### DISCUSSION THREAD

**Discussion by Karim-Mane** [epiverse-trace/epiverse-trace.github.io#47] [Project Ideas]
### Description

...
We aim at implementing some automated data cleaning procedures to help the users:

* check data format in specific column
* check for possible mistakes during data entry
* convert and deal with date column
* check and deal with outliers and unwanted data
* correct erroneous data 

### Typical end-users

R users, epidemiologists

### Potential contributors

Karim Mané; Joshua Lambert

### Key collaborators

LSHTM

### Inputs

data frame with relevant information

### Outputs

a cleaned and checked data frame

### Imports

* <*janitor*>
* <*stringr*>
* <*dplyr*>
* <*data.table,*>
* <*lubridate*>
* <*naniar*>
* <*readr*>

### Used by

* <*any package in epiverse that takes a data frame as input*>
* <*second package it could export data to*>
* ...

### Related projects

* <*a package doing related things *>
* <*another package doing related things *>
* ...

### Usage

```bash
...
```

### Additional comments

...

**Comment by pacoramon** [comment]
Hi, I think it would be great if you could also check the R packages: 
- [validate](https://cran.r-project.org/web/packages/validate/vignettes/cookbook.html)
- [data.validation](https://github.com/Appsilon/data.validator)
- [dataquieR](https://dataquality.ship-med.uni-greifswald.de/IntroductoryTutorial.html)

to check for possible mistakes during data entry. 

Still determining if this is part of what you have in mind, or if you may feel that this could correspond to a later step in the data management process before analysis, but it would t might be worth looking into it. 

Best,

**Comment by Bisaloo** [comment]
Development is now ongoing at https://github.com/epiverse-trace/cleanepi. Please add ideas, feature requests, etc. there. Thanks!
